### ISSUE TITLE
Inconsistent Image Normalization between Training and Inference (Pi3X)

### ISSUE THREAD

**Issue by steveli88** [yyfz/Pi3#146]
Hi @yyfz,

Thank you for the impressive work on Pi3X! I am writing to report a discrepancy I found regarding image preprocessing that significantly impacts performance on dynamic subjects.

### Problem Description
While using the Pi3X official implementation to generate point clouds from video (with multimodal input disabled), I noticed a significant performance gap:
**Buildings (Static)**: Accurate point cloud reconstruction.
**Human subjects (Slight move)**: Poor/distorted predictions.

### Investigation & Findings
After comparing the `main` branch (inference) with the `train` branch, I identified an inconsistency in image normalization:
**Inference**: The code includes an explicit image normalization step before the forward pass. https://github.com/yyfz/Pi3/blob/b412c3bd236dfd7686f1e4b48004d5087f2fa093/pi3/models/pi3x.py#L247
**Training**: The training pipeline appears to omit this normalization step.

### Results after Modification
I tested removing the image normalization from the inference script to match the training logic. This change resolved the issue, resulting in significantly improved point cloud quality for human subjects.

**With normalization step**
<img width="1920" height="1079" alt="Image" src="https://github.com/user-attachments/assets/c56fe3dd-16ea-481a-a820-3b1614bf9eaf" />

**Without normalization step**
<img width="1920" height="1079" alt="Image" src="https://github.com/user-attachments/assets/f26dc690-0939-4809-8320-6b4d9e0c86e3" />

### Questions/Insights
Was Pi3X intentionally trained without the image normalization (e.g., mean/std scaling)?
Should the inference code in the main branch be updated to remove this step, or is there a specific use case where normalization is still required?

I would appreciate your insights. Thank you!

**Comment by yyfz** [comment]
Hi, [steveli88](https://github.com/steveli88)!

It seems like this is a special case regarding the normalization issue. We always normalize the input RGB values to match the DINOv2 input distribution. Therefore, the best results should generally require normalization; your specific case might be an exception.

**Comment by steveli88** [comment]
Hi @yyfz ,

Thank you very for your fast response. 
The dirty Pi3X training code exclude the normalization step, 
https://github.com/yyfz/Pi3/blob/fc93bd350d97dc09a402021305e0c0a2ec9316f9/pi3/models/pi3x_dirty_training/pi3x_training.py#L272
do I need to add it back if I want to finetune Pi3X?

**Comment by yyfz** [comment]
Yes, you should add it back!

**Comment by steveli88** [comment]
Thank you.
Also, I test Pi3X on our provide evaluation metric, Pi3X with and without normalization only different slightly. Pi3X with normalization is slightly better in most metrices.
